channeldb/migrations.go

```diff
@@ -226,12 +231,28 @@ func migrateInvoiceTimeSeries(tx *bbolt.Tx) error {
 			return err
 		}
 
-		return invoices.Put(invoiceNum, b.Bytes())
+		// Make a copy of the key since it is only valid within the
+		// function closure.
```

Comment:
Why is this the case?

Reply:
https://github.com/etcd-io/bbolt#foreach
> Please note that keys and values in ForEach() are only valid while the transaction is open. If you need to use a key or value outside of the transaction, you must use copy() to copy it to another byte slice.

Reply:
Doesn't seem necessary then since everything's happening within a single database transaction.

Reply:
mhm see my other comment

Reply:
You're right. I read the documentation as only valid within the ForEach loop, but it is indeed the database tx itself. Fixed: https://github.com/lightningnetwork/lnd/pull/3198/commits/5a4057978d6e34d9d2bdad18a8358db95854e102